tests/index.js

```diff
@@ -0,0 +1,20 @@
+const QUnit = require('qunit');
```

Comment:
I'm actually not sure if you need this since you are using the `qunit` binary to run the tests.

Reply:
Seems to be necessary.